template.conf

```diff
@@ -43,14 +43,17 @@
       include "{{ settings.overrides }}/{{ site.shortname }}/server";
 
       {# We need a named location block in order to use try_files. #}
-      {% if site.backends.length %}
+
+      {% set defaultBackendGroup = site.backendGroups[0] %}
+      {% set proxyRoot = defaultBackendGroup and (defaultBackendGroup.path == '/') %}
+      {% if proxyRoot %}
         location @proxy-{{ site.shortname }}-{{ options.port }} {
           {% if site['https-upstream'] %}
-            proxy_pass https://upstream-{{ site.shortname }};
+            proxy_pass https://upstream-{{ site.shortname }}-1;
```

Comment:
Unclear why this changed.

Reply:
We now have multiple upstreams, one per backend group. A backend group is the collection of backends that are interested in a particular path prefix. Because of the way the backends are sorted, any backend group responding for all paths will be the first in the array and thus get the "-1" suffix (see use of loop.index elsewhere in this file). This is the one that gets the actual-website treatment.